src/EnergyPlus/FuelCellElectricGenerator.cc

```diff
@@ -1330,6 +1331,20 @@ namespace FuelCellElectricGenerator {
                                 "System",
                                 "Sum",
                                 this->Name);
+
+            SetupOutputVariable(state, "Generator Number of Cycles",
+                                OutputProcessor::Unit::None,
+                                this->Report.NumCycles,
+                                "System",
+                                "Average",
+                                this->Name);
+
+            SetupOutputVariable(state, "Generator Power Module Skin Heat Loss Rate",
+                                OutputProcessor::Unit::W,
+                                this->Report.FCPMSkinLoss,
+                                "System",
+                                "Average",
+                                this->Name);
```

Comment:
Register two new **Advanced** report output variables to test changes for #8513 and #8514 

You need this to be able to see them:

```
  Output:Diagnostics,
    DisplayAdvancedReportVariables;
```